### PR TITLE
fix: migrate to module ES6

### PR DESCRIPTION
**Pull request by josoriom**

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Node.js CI
```
Comment (targos) [inline .github/workflows/nodejs.yml:2]: please use the latest version of the workflow from https://github.com/mljs/.github/blob/master/workflow-templates/nodejs.yml

File: src/measures.js [hunk 2]
```diff
@@ -1,138 +1,151 @@
-'use strict';
-
 // Accuracy
-exports.acc = pred => {
```
Comment (targos) [inline src/measures.js:4]: `export function acc(pred) { ... }`

Same for all other functions here

File: src/index.js [hunk 3]
```diff
@@ -1,227 +1,238 @@
...
-                    });
-                }
-            }
-        }
-
-        if (isDistance) {
-            predP.sort((a, b) => a.pred - b.pred);
-        } else {
-            predP.sort((a, b) => b.pred - a.pred);
+import measures from './measures';
```
Comment (targos) [inline src/index.js]:
```suggestion
import * as measures from './measures';
```

**Comment by targos** [comment]
Have you changed something in the algorithms apart from `var->let` ?

**Comment by josoriom** [comment]
No, I made a simple test and It works for me.

File: .github/workflows/nodejs.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+
+name: Node.js CI
+
+on:
+  push:
+    branches: $default-branch
```
Comment (targos) [inline .github/workflows/nodejs.yml]:
```suggestion
    branches: master
```

File: src/index.js [hunk 5]
```diff
@@ -1,227 +1,239 @@
...
-        }
-        var auc = 0;
-        for (i = 1; i < l; i++) {
-            auc += 0.5 * (x[i] - x[i - 1]) * (y[i] + y[i - 1]);
-        }
-        return auc;
+    this.nPos = nPos;
+    this.nNeg = nNeg;
+    this.nSamples = nPos + nNeg;
+    this.measures = measures;
```
Comment (targos) [inline src/index.js]: why?

**Pull request merged**